lib/cfd_trading/cfd_offer.dart

```diff
@@ -56,8 +56,6 @@ class _CfdOfferState extends State<CfdOffer> {
     final margin = Amount(250000).display(currency: Currency.btc).value;
     // TODO: calculate expiry
     final expiry = DateFormat('dd.MM.yy-kk:mm').format(DateTime.now());
-    // TODO: calculate funding rate
-    final fundingRate = Amount(200).display(currency: Currency.btc).value;
```

Comment:
That's interesting. Imho we still need to charge some form of a _fee_ for the time the position is open. No need to action this for now. Just want to bring it up.

Reply:
Oh yeah, we still want to charge a fee. But for this early version I think it just makes things a bit more complicated than necessary.